notebooks/tou/hpwh_schedule_basic.qmd

```diff
@@ -0,0 +1,175 @@
+---
+title: "First-Pass TOU Scheduling Decision Model for HPWHs in OCHRE"
+author: "Switchbox"
+date: "2025-06-13"
+format:
+  html:
+    mathjax: true
+---
+
+# Introduction
+
+This document outlines a simple, first-pass heuristic-based decision model for consumer response to time-of-use (TOU) electricity rates in residential building simulations using OCHRE, focusing on Heat Pump Water Heaters (HPWHs). The goal is to integrate an adaptive control schedule for the HPWH based on feedback from utility bills and anticipated cost savings, while accounting for consumer effort in shifting schedules.
+
+# 1. Problem Definition
```

Comment:
@alexhyunminlee @sherryzuo You need a sentence or two at the start of every h1 section that explains what the section is about and how it ties in with the rest. You need to thread the sections together with a narrative.